pepper-apis/docs/specification/src/components/responses.yml

```diff
@@ -58,7 +58,10 @@ ActivityAnswersPatchBadPayloadResponse:
       schema:
         $ref: '../pepper.yml#/components/schemas/Error.ActivityAnswersPatchBadPayload'
 ActivityAnswersPatchInvalidResponse:
-  description: activity instance or question is read-only or answer submission failed validation
```

Comment:
@bskinner I updated the documentation descriptions to better spell out parent/child behavior.